astropy/coordinates/builtin_frames/galactocentric.py

```diff
@@ -46,29 +52,36 @@ class Galactocentric(BaseCoordinateFrame):
         {\rm Dec} = -28:56:10.23~{\rm deg}
 
     The default distance to the Galactic Center is 8.3 kpc, e.g.,
-    Gillessen et al. 2009,
-    http://adsabs.harvard.edu/abs/2009ApJ...692.1075G.
+    Gillessen et al. (2009),
+    https://ui.adsabs.harvard.edu/#abs/2009ApJ...692.1075G/abstract
```

Comment:
Nitpick, but why the longer form of the URL? I'd think the older one may be the more stable one...